ui/css/perf.css

```diff
@@ -31,14 +30,6 @@ to { opacity: 0; }
   box-sizing: content-box;
 }
```

Comment:
.tt-query and .tt-hint belong to typeahead too. Could you take out this entire block, lines 19-32?